pagopa/io-app
# feat: [#178212417] Feature profile data on one screen

# THIS PR DEPENDS ON #3082 

## Short description
Moved SPID email from preferences screen to profile data screen, deleted useless SPID phone number, added name and surname, changed some layout.

## How to test
Press "Your data" in profile main screen and see if is equal to Figma [Click here](https://www.figma.com/file/az44bKOncugExb9FxTEoLg/%23178212417?node-id=306%3A2261)

**Before:**
<img width="300" alt="Screenshot 2021-06-04 at 17 23 53" src="https://user-images.githubusercontent.com/61834253/120825459-b38c2700-c559-11eb-9828-bd8fc7d5f0fe.png">

**After:**
<img width="300" alt="Screenshot 2021-06-04 at 17 25 05" src="https://user-images.githubusercontent.com/61834253/120825616-d9193080-c559-11eb-8858-384dc7c0af1d.png">

<img width="300" alt="Screenshot 2021-06-04 at 17 25 42" src="https://user-images.githubusercontent.com/61834253/120825700-ef26f100-c559-11eb-9804-1c7e364ac674.png">
